.github/workflows/build-and-deploy-preprod.yml

```diff
@@ -0,0 +1,55 @@
+name: Build-Deploy
+on:
+  push:
+    branches:
+      - '*'
+      - '*/*'
+env:
+  IMAGE: docker.pkg.github.com/${{ github.repository }}/familie-ba-mottak:${{ github.sha }}
```

Comment:
Kun sha er en endring fra eksisterende format dato-sha (2020.01.09-a854304). Er dette ønsket oppførsel?

Reply:
For nå er det greit for meg - inntil jeg finner en mer lettlest måte å injecte datoen på.